### PR TITLE
add new endpoint POST /taas-teams/:id/members

### PR DESCRIPTION
**Pull request by imcaizheng**
## Verification
- Run `taas-apis` locally.
- Fire the `POST /taas-teams/:id/members` request in Postman. The response would look like:

    ``` json
    {
        "success": [
            {
                "id": 14159,
                "userId": 40159097,
                "role": "customer",
                "createdAt": "2021-02-19T08:06:20.392Z",
                "updatedAt": "2021-02-19T08:06:20.392Z",
                "createdBy": -101,
                "updatedBy": -101,
                "handle": "tester1234",
                "photoURL": null,
                "workingHourStart": "9:00",
                "workingHourEnd": "17:00",
                "timeZone": "Asia/Kolkata",
                "email": "sathya.jayabal@gmail.com"
            },
            {
                "id": 14160,
                "userId": 1800091,
                "role": "customer",
                "createdAt": "2021-02-19T08:06:20.526Z",
                "updatedAt": "2021-02-19T08:06:20.526Z",
                "createdBy": -101,
                "updatedBy": -101,
                "handle": "Scud",
                "photoURL": "https://topcoder-dev-media.s3.amazonaws.com/member/profile/Scud-1450982908556.png",
                "email": "email@domain.com",
                "workingHourStart": null,
                "workingHourEnd": null,
                "timeZone": null
            }
        ],
        "failed": [
            {
                "error": "User doesn't exist",
                "handle": "non-existing"
            },
            {
                "error": "User doesn't exist",
                "email": "non-existing@domain.com"
            }
        ]
    }
    ```

    Notice that two members are added to the project.

- Repeat the request once. The response would look like:

    ``` json
    {
        "success": [],
        "failed": [
            {
                "error": "User doesn't exist",
                "handle": "non-existing"
            },
            {
                "error": "User doesn't exist",
                "email": "non-existing@domain.com"
            },
            {
                "error": "User is already added",
                "handle": "tester1234"
            },
            {
                "error": "User is already added",
                "email": "email@domain.com"
            }
        ]
    }
    ```

    Notice that the two members previously added to the project failed to be added to the project again.

### CODE REVIEW

**Review by maxceem — CHANGES_REQUESTED** [review]
Thank you, @imcaizheng works great.

Only one thing, we should allow calling this endpoint to any user who has access to the team, not only to the team members.

In other words, anyone who can `GET /taas-teams/:id` should be able to `POST /taas-teams/:id/members`.

File: src/services/TeamService.js [hunk 1]
```diff
@@ -373,7 +373,7 @@ async function _addMemberToProjectAsCustomer (projectId, userId) {
  * @returns {Object} the success/failed added members
  */
 async function addMembers (currentUser, id, data) {
-  await helper.checkIsMemberOfProject(currentUser.userId, id)
+  await helper.getProjectById(currentUser, id) // check whether the user can access the project
```
Comment (imcaizheng) [inline src/services/TeamService.js:376]: @maxceem Code updated.

File: app.js [hunk 2]
```diff
@@ -75,8 +75,8 @@ app.use((err, req, res, next) => {
   }
 
   if (err.response) {
-    // extract error message from V3 API
-    errorResponse.message = _.get(err, 'response.body.result.content')
+    // extract error message from V3/V5 API
+    errorResponse.message = _.get(err, 'response.body.result.content') || _.get(err, 'response.body.message')
```
Comment (imcaizheng) [inline app.js:79]: I made a slight improvement here by the way.

Comment (maxceem) [reply app.js:79]: yeah, this change looks good

**Review by maxceem — APPROVED** [review]
Thank you @imcaizheng works perfectly now.

**Pull request merged**